jdbcSource/README.md

```diff
@@ -104,6 +107,12 @@ The Configuration document may look similar to the following example:
 *   **username**: Required. This is the username that will be used to connect to the SQL Database.
 *   **password**: Required. This is the password that will be used to connect to the SQL Database.
 *   **dbURL**: Required. This is the URL corresponding to the SQL Database that you will connect to.
+*   **asynchronousProcessing**: Optional. If set to `true`, query and publish requests will be handled asynchronously, as 
+opposed to the default behavior which is synchronous.
+*   **maxRunningThreads**: Optional. Only used if `asynchronousProcessing` is set to `true`. The maximum number of threads 
```

Comment:
Would a better name be something like `maxActiveTasks`?  Then it's parallel to `maxQueuedTasks` and can describe what's actually happening.  "I can queue up some things, but only n of them can be active at the same time"